research/picai/fedavg/README.md

```diff
@@ -1,29 +1,15 @@
 # Running FedAvg Example
 
-The following instructions outline training and validating a simple U-Net model on the Preprocessed PICAI Dataset described in the [PICAI Documentation](/research/picai/README.md) in a federated manner across two clients using FedAvg. The dataset is partitioned randomly in a uniform manner based on the number of clients. Two scripts are provided, one in which the server and clients are spun up on the same machine and one in which the server and each serperate client is spun up on a different machine along with example invocations below. A python environment with the required libraries must already exist.  See the main PICAI documentation Cluster [PICAI Documentation](/research/picai/README.md) for instructions on creating and activating environment required to exectute the following code.
+The following instructions outline training and validating a simple U-Net model on the Preprocessed PICAI Dataset described in the [PICAI Documentation](/research/picai/README.md) in a federated manner across two clients using FedAvg. The dataset is partitioned randomly in a uniform manner based on the number of clients. The provided script spins up server and clients on the same machine which is demonstrated below. The script will automatically handle relaunching jobs that timeout. A python environment with the required libraries must already exist and a path to a configuration file must be specified.  See the main [PICAI Documentation](/research/picai/README.md) for instructions on creating and activating the environment required to exectute the following code.
 
-## Server and Client on Seperate Machines
```

Comment:
Is this option no longer available/?

Reply:
I removed it because I think it just makes a lot more sense to run in a single job, especially considering the length of these experiments. If submitted within a single job we can now do the checkpointing and resubmit automatically but with different jobs for clients and server its going to be really hard to orchestrate

Reply:
Yeah that makes sense. If everything fits already in memory etc. then we can do single node training for now.

Reply:
To get around this, I think we can write a slurm script that requests the job all at once and then distributes resources with srun. That way if we get pre-empted, its all at once. As opposed to request the resources in several jobs. I will look into adding this for the next PR